apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/dashboard/_components/BatchEvaluationsTable.tsx

```diff
@@ -62,7 +62,7 @@ export default function BatchEvaluationsTable({
                     .detail({ id: project.id })
                     .commits.detail({ uuid: commit.uuid })
                     .documents.detail({ uuid: document.documentUuid })
-                    .evaluations.detail(evaluation.uuid).dashboard.destroy
+                    .evaluations.detail(evaluation.id).destroy
```

Comment:
What is this `/dashboard` prefix? I think is not necessary no?

Reply:
talked offline